simulator-ui/tests/helpers/helper-functions.ts

```diff
@@ -0,0 +1,94 @@
+import { expect, Page } from '@playwright/test';
+import { navbarElementLinkPair } from './helper-interfaces';
+
+export const elementLinkPairNoDropdown: navbarElementLinkPair[] = [
+  { testName: 'navigationScenariosLink', link: /.*scenario*/, apiLink: '**/api/scenarios*' },
+  { testName: 'navigationScenarioExecutionsLink', link: /.*scenario-result*/, apiLink: '**/api/scenario-executions*' },
+];
+export const elementLinkPairEntityDroptdown: navbarElementLinkPair[] = [
```

Comment:
```suggestion
export const elementLinkPairEntityDropdown: navbarElementLinkPair[] = [
```

Reply:
I restructured it in hope, it will be better understandable: With parent navbar elements who can have child elements.